easycv/datasets/detection/data_sources/pai_format.py

```diff
@@ -113,18 +113,18 @@ def parser_manifest_row_str(row_str):
                 bnd = [x, y, x + w, y + h]
                 class_name = obj['labels'][0]
                 bboxes.append(bnd)
-                class_names.append(class_name)
+                gt_labels.append(classes.index(class_name))
         break
 
-    parse_results['gt_bboxes'] = bboxes
-    parse_results['class_names'] = class_names
     parse_results['filename'] = img_url
+    parse_results['gt_bboxes'] = np.array(bboxes, dtype=np.float32)
+    parse_results['gt_labels'] = np.array(gt_labels, dtype=np.int64)
```

Comment:
gt_labels is not a good name, consider use gt_bbox_class_ids gt_bbox_class_names

Reply:
consistent with the field of transforms

Reply:
ok